lib/install/validate/validator.go

```diff
@@ -902,3 +914,28 @@ func (v *Validator) syslog(op trace.Operation, conf *config.VirtualContainerHost
 		RAddr:   host,
 	}
 }
+
+// FIXME ATC DEBT setting this value needs to be moved to Dispatcher
+// https://github.com/vmware/vic/issues/6803
+// set PersistNetworkBacking key to "true"
+func (v *Validator) ConfigureVCenter(ctx context.Context) error {
+	op := trace.FromContext(ctx, "Set vCenter serial port backing")
+	defer trace.End(trace.Begin("", op))
+
+	errMsg := "Set vCenter settings SKIPPED"
+	if !v.sessionValid(op, errMsg) {
+		return nil
+	}
+	if !v.IsVC() {
+		op.Info(errMsg)
+		return nil
+	}
+
+	err := optmanager.UpdateOptionValue(ctx, v.Session, persistNetworkBackingKey, "true")
```

Comment:
I thought that we found we needed to omit `config` from the key for it to be set, and that prefix magically got appended, or was it something else that was inhibiting it from being updated?

Reply:
that was reading the key when it was unset both with and without `config` would give an error so now we assume that if there's an error it's unset and go ahead and set the value